bitnami/cilium/templates/configmap.yaml

```diff
@@ -0,0 +1,119 @@
+pprof-port: {{ printf "%d" (int .Values.agent.containerPorts.pprof) | quote }}
+{{- end }}
+# Monitor aggregation
+monitor-aggregation: medium
+monitor-aggregation-interval: "5s"
+monitor-aggregation-flags: all
+# BPF configuration
+bpf-root: /sys/fs/bpf
+preallocate-bpf-maps: "false"
+# CNI configuration
+cni-exclusive: "true"
+custom-cni-conf: "false"
+cni-log-file: "/opt/bitnami/cilium/var/run/cni.log"
+write-cni-conf-when-ready: {{ printf "/host%s/05-cilium.conflist" .Values.agent.cniPlugin.hostCNINetDir }}
+cni-uninstall: {{ ternary "true" "false" .Values.agent.cniPlugin.uninstall | quote }}
+# Operator configuration
+operator-api-serve-addr: {{ printf ":%d" (int .Values.operator.containerPorts.api) | quote }}
+disable-endpoint-crd: "false"
+skip-crd-creation: "false"
+identity-allocation-mode: crd
+ipam: "cluster-pool"
+cluster-pool-ipv4-cidr: "10.0.0.0/8"
+cluster-pool-ipv4-mask-size: "24"
+# Hubble configuration
+enable-hubble: "true"
+hubble-socket-path: "/opt/bitnami/cilium/var/run/hubble.sock"
+hubble-export-file-max-size-mb: "10"
+hubble-export-file-max-backups: "5"
+hubble-listen-address: {{ printf ":%d" (int .Values.agent.containerPorts.hubblePeer) | quote }}
+hubble-disable-tls: {{ ternary "false" "true" .Values.tls.enabled | quote }}
+{{- if .Values.tls.enabled }}
+hubble-tls-cert-file: /certs/hubble/tls.crt
+hubble-tls-key-file: /certs/hubble/tls.key
+hubble-tls-client-ca-files: /certs/hubble/ca.crt
+{{- end }}
+{{- if or .Values.agent.metrics.enabled .Values.operator.metrics.enabled }}
+# Prometheus metrics
+enable-metrics: "true"
+{{- if .Values.agent.metrics.enabled }}
+prometheus-serve-addr: {{ printf ":%d" (int .Values.agent.containerPorts.metrics) | quote }}
+hubble-metrics-server: {{ printf ":%d" (int .Values.agent.containerPorts.hubbleMetrics) | quote }}
+controllerGroupMetrics: all
+metrics: ~
+hubble-metrics: ~
+{{- end }}
+{{- if .Values.operator.metrics.enabled }}
+operator-prometheus-serve-addr: {{ printf ":%d" (int .Values.operator.containerPorts.metrics) | quote }}
+{{- end }}
+{{- end }}
+# Other configuration
+enable-k8s-networkpolicy: "true"
+synchronize-k8s-nodes: "true"
+remove-cilium-node-taints: "true"
+set-cilium-node-taints: "true"
+set-cilium-is-up-condition: "true"
+agent-not-ready-taint-key: "node.cilium.io/agent-not-ready"
+{{- end }}
+{{- end }}
+
+{{- if not .Values.existingConfigmap }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "common.names.fullname" . }}
+  namespace: {{ include "common.names.namespace" . | quote }}
+  labels: {{- include "common.labels.standard" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 4 }}
+    app.kubernetes.io/part-of: cilium
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+data: {{- mergeOverwrite (include "cilium.configuration" . | fromYaml) .Values.overrideConfiguration | toYaml | nindent 2 }}
```

Comment:
Could it be possible to allow templating in the overrideConfiguration just like you are allowing with configuration?

Reply:
done at https://github.com/bitnami/charts/pull/26374/commits/c5ee18dd06171ea3c925861c0811632c8cf7281c